test/lint/lint-includes.py

```
@@ -47,7 +40,7 @@ def get_toplevel():
 
 
 def list_files_by_suffix(suffixes):
-    exclude_args = [":(exclude)" + dir for dir in EXCLUDED_DIRS]
+    exclude_args = [":(exclude)" + dir for dir in SHARED_EXCLUDED_SUBTREES]
```

Ruff (0.15.15)

[error] Variable `dir` is shadowing a Python builtin

(A001)